largest-number-from-array.playground/Contents.swift

```diff
@@ -17,5 +15,5 @@ XCTAssertEqual(90, largestNumberComposed(from: [9,0]))
 
 XCTAssertEqual(910, largestNumberComposed(from: [10,9]))
 
-XCTAssertEqual(95021, largestNumberComposed(from: [50,2,1,9]))
+XCTAssertEqual(56554, largestNumberComposed(from: [54,5,56]))
```

Comment:
As you have pointed out, a lexicographic sort and concatenation is close, but not quite correct. For example, for the numbers 5, 54, and 56 a lexicographic sort will produce {5, 54, 56} (in increasing order) or {56, 54, 5} (in decreasing order), but what we really want is {56, 5, 54}, since that produces the largest number possible.